app/controllers/AuthenticationController.scala

```diff
@@ -429,6 +453,58 @@ class AuthenticationController @Inject()(
     }
   }
 
+  lazy val absoluteCallbackURL = s"${conf.Http.uri}/api/auth/oidc/callback"
+
+  def loginViaOpenIdConnect(): Action[AnyContent] = sil.UserAwareAction.async { implicit request =>
+    openIdConnectClient.getRedirectUrl(absoluteCallbackURL).map(url => Redirect(url))
+  }
+
+  def loginUser(user: User, loginInfo: LoginInfo)(implicit request: Request[AnyContent]): Future[Result] =
```

Comment:
looks like the passed user is not used (a new variable by the same name is immediately fetched using loginInfo). In this case, I’d remove the user argument